crates/miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -200,16 +200,9 @@ end
 #! Where:
 #! - ACCOUNT_COMMITMENT is the commitment of the account data.
 export.compute_current_commitment
-    exec.memory::is_native_account
-    # => [is_native_account]
-
-    # first compute the latest storage commitment
-    # since foreign accounts do not change during transaction execution, this step can be skipped
-    # for them
-    if.true
-        exec.compute_current_storage_commitment
-        # => []
-    end
```

Comment:
Do we no longer need to check this? Should we use the same methodology here as with the storage commitment? (i.e., get something like `should_recompute_account_commitment` from the `memory` module)

Reply:
No, these checks became redundant, since we are essentially checking the same inside the `recompute_storage_commitment` procedure. But I'll update the comment here to explain that.